UI/SitRepPanel.cpp

```diff
@@ -269,7 +265,7 @@ namespace {
             AttachChild(m_icon);
 
             m_link_text = new LinkText(GG::X0, GG::Y0, GG::X1, m_sitrep_entry.GetText() + " ", 
-                                       ClientUI::GetFont(std::max(1.0*ClientUI::Pts(), 1.0*GetOptionsDB().Get<int>("UI.sitrep-font-size"))),
+                                       ClientUI::GetFont(1.0*ClientUI::Pts()),
```

Comment:
no need for 1.0* or any parameter to GetFont... the no-parameter version defaults to the same thing.